### PR TITLE
Release 0.1.1

### PR DESCRIPTION
**Pull request by pavish**
Release 0.1.1

Related to https://github.com/centerofci/mathesar/issues/2705

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `develop` branch of the repository
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: mathesar_ui/src/component-library/confirmation/Confirmation.svelte [hunk 1]
```diff
@@ -34,6 +34,7 @@
     } catch (error) {
       // @ts-ignore: https://github.com/centerofci/mathesar/issues/1055
       onError(error);
+      modal.close();
```
Comment (pavish) [inline mathesar_ui/src/component-library/confirmation/Confirmation.svelte:37]: The modal seems to close on an error which may not want in several circumstances.

This comes from a community PR: https://github.com/centerofci/mathesar/pull/2632

We need to look at this change.

File: mathesar_ui/src/systems/data-explorer/QueryRunner.ts [hunk 2]
```diff
@@ -83,6 +83,7 @@ export default class QueryRunner<
     void this.run();
     this.selection = new SheetSelection({
       getColumns: () => [...get(this.processedColumns).values()],
+      getColumnOrder: () => [], // Empty array to default to the columnIndex order
```
Comment (pavish) [inline mathesar_ui/src/systems/data-explorer/QueryRunner.ts:86]: This prevents multi column/cell selection.

Needs to be reverted.

File: mathesar_ui/src/component-library/confirmation/Confirmation.svelte (hunk 1 above)
Comment (seancolsen) [reply mathesar_ui/src/component-library/confirmation/Confirmation.svelte:37]: Thanks for noticing this, @pavish. I've opened #2729 to fix this.

**Comment by seancolsen** [comment]
@pavish I looked through the diff and I didn't see any other red flags 🤞🏻

**Comment by rajatvijay** [comment]
@pavish I also skimmed through, no red flags.

**Comment by pavish** [comment]
Thanks @seancolsen and @rajatvijay. 
For the backend changes, @mathemancer already gave me the go signal on this via matrix chat.

**Comment by pavish** [comment]
I'll keep this PR open. I'll update the description when we're ready to merge 0.1.1 into master.

**Pull request merged**